src/components/enter-phone-number/tests/enter-phone-number-integration.test.ts

```diff
@@ -73,7 +73,7 @@ describe("Integration::enter phone number", () => {
       .expect(function (res) {
         const $ = cheerio.load(res.text);
         expect($("#phoneNumber-error").text()).to.contains(
-          "Enter a UK phone number"
+          "Enter a mobile phone number"
```

Comment:
I think it's this one failing? I think it should just be expecting "Enter a phone number"

Reply:
fixed